src/WingetCreateTests/WingetCreateTests/TestUtils.cs

```diff
@@ -55,6 +56,16 @@ public static void SetMockHttpResponseContent(string installerName)
             content.Headers.ContentType = new MediaTypeHeaderValue("application/octet-stream");
             httpResponseMessage.Content = content;
             PackageParser.SetHttpMessageHandler(httpMessageHandler);
+        }
+
+        /// <summary>
+        /// Obtains the relative filepath of the resources test data directory.
+        /// </summary>
+        /// <param name="fileName">File name of the test file.</param>
+        /// <returns>Full path of the test file.</returns>
+        public static string GetTestFile(string fileName)
+        {
+            return Path.Combine(Environment.CurrentDirectory, @"Resources\", fileName);

```

Comment:
>\ [](http://example.com/codeflow?start=73&length=1)

no biggie but for future reference you don't need the trailing slash here

Reply:
i knew that 😅. Removed extra trailing slash.